dev/source/docs/building-the-code.rst

```diff
@@ -15,18 +15,42 @@ Setting up the Build Environment
 - :ref:`Setup the Build Environment on Windows <building-setup-windows>`
 - :ref:`Setup the Build Environment on MacOSX <building-setup-mac>`
 
+
 Building / Compiling
 --------------------
 
-**Linux / MacOSX users:**
+Once the build environment is set up as above, run the following commmands prior to the initial build. (For Windows users, this is using WSL, as set up above):
 
-- Linux and MacOSX users should build with waf as described in `BUILD.md <https://github.com/ArduPilot/ardupilot/blob/master/BUILD.md>`__.
+- `git clone https://github.com/ArduPilot/ardupilot`
+- `cd ardupilot`
+- `git submodule update --recursive --init`
```

Comment:
we shouldn't need this as previous instruction have done it already

Reply:
A link to those instructions here would suffice.

Reply:
I disagree.

Somewhere on the welcome to dev with ardupilot we must show how to use the basics of waf.

That little link to the giant list in build.md is easy to miss and has too much detail...
We need a cohesive easy to follow story. 

I originally figured out how to use waf because I watched the commands  the ArduPilot extension was producing.